### ISSUE TITLE
[ENHANCEMENT]: Show more info in the dashboard

### ISSUE THREAD

**Issue by ZonPV** [edterbak/NodeRed_Heishamon_control#174]
_**Is your feature request related to a problem or use case? Please describe.**_
I would like to see more info in the dashboard. Missing info about temperature and energy.

_**Describe the solution you'd like**_
A value about the Room temperature and Outside temperature as well as the Power, Consumption in Watt at that moment and if possible also the usage of energy today

_**Describe alternatives you've considered**_
Those values are now available in Domoticz.. I have to look them up in other app.

It is just for fun.. No hurry either. Thanks in advance.

**Comment by WP-Rue** [comment]
You can easily build your own additional dashboard side to show whatever you like..
So my start page as possibility..:

<img width="1420" alt="Image" src="https://github.com/user-attachments/assets/941f5c5f-fd08-49a8-a973-49f4720e4342" />

**Comment by ZonPV** [comment]
Yes, I know. But what about your flow when you update the original flows from Ed? In my situation I added extra nodes in WP-dash. That can be a lot of work. How did you solve that problem?

![Image](https://github.com/user-attachments/assets/ed457d85-ed39-47b8-8c51-4e6544d68346)

**Comment by WP-Rue** [comment]
Ed re recommends an extra tab or in Nodered it is called flow for placing the personal stuff- and so I do. So don't put your extra display nodes on the wp-dash, because you have to delete this for update but create your own flow for this.
For getting the information in this extra flow I mostly use mqtt-in and get the information directly from Heishamon- so completely independent from Ed's flow- so update resistent.
We can also use the Link-in Node Ed is providing for external controls- e.g. setting DHW setpoint, providing an external outside temperature sensor...
For some function you can refer to global variables, that Ed is using:
I switch on or off one schedule event for night pause heating in dependence of mean outside temperature..
Everything is possible 😉

**Comment by edterbak** [comment]
I think this issue can be closed, right?

If not, please reopen it again with your current question
THank you 😄
